pkg/model/lattice/iamauthpolicy.go

```diff
@@ -0,0 +1,60 @@
+package lattice
+
+import (
+	"fmt"
+
+	anv1alpha1 "github.com/aws/aws-application-networking-k8s/pkg/apis/applicationnetworking/v1alpha1"
+	"github.com/aws/aws-application-networking-k8s/pkg/utils"
+	"github.com/aws/aws-sdk-go/aws/arn"
+)
+
+type AuthPolicyKind string
+
+const (
+	GatewayAuthPolicy AuthPolicyKind = "gateway"
+	ServiceAuthPolicy AuthPolicyKind = "service"
```

Comment:
Isn't it Gateway, HTTPRoute, and GRPCRoute? Service is the VPC Lattice resource

Reply:
Also, if we are using the K8s Kinds, it would be good to capitalize them so they can be safely compared with other Kind strings

Reply:
Should be LatticeServiceNetworkAuthPolicy and LatticeServiceAuthPolicy?